festerize.py

```diff
@@ -217,24 +218,37 @@ class FesterizeError(IntEnum):
 
             # Handle the response.
             if r.status_code == 201:
-                # Send an awesome message to the user.
-                border_char = extra_satisfaction[
-                    random.randint(0, len(extra_satisfaction) - 1)
-                ]
-                border_length = 2 + (20 + len(csv_filename)) // 2
+                click.echo("Uploaded {} successfully".format(csv_filename))
 
-                click.echo(border_char * border_length)
+                # Check the returned CSV
+                content_length_header_key = "Content-Length"
                 click.echo(
-                    "{} SUCCESS! Uploaded {} {}".format(
-                        border_char, csv_filename, border_char
+                    "{}: {}".format(
+                        content_length_header_key, r.headers[content_length_header_key]
                     )
                 )
-                click.echo(border_char * border_length)
 
                 # Save the result CSV to the output directory.
-                out_file = click.open_file(os.path.join(out, csv_filename), "wb")
-                out_file.write(r.content)
-                out_file.close
+                with open(os.path.join(out, csv_filename), "wb") as f:
+                    num_bytes_written = f.write(r.content)
```

Comment:
Good to close the file handle when you're done with it(?)

Reply:
That's what the `with` construct is for; it's like Java's try-with-resources

Reply:
More info: https://docs.python.org/3/reference/compound_stmts.html#the-with-statement

Reply:
Ah, got it. Thanks.